wikimedia/wikipedia-ios
# Image recommendations - localization fixes

**Phabricator:** https://phabricator.wikimedia.org/T363925

### Notes
This fixes a couple of bugs with localizations. "Advanced settings" was never exported to translatewiki, so this updates the extract script to include that subdirectory. I also added a missing line in the email prompts.

### Test Steps
1. Tap "Problem with feature" in overflow menu. Confirm "[Screenshots or Links]" is added at the end.
